week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,52 @@
+//----------------PART-ONE-----------------------------:
+
+let blueButton = document.getElementById("blueBtn");
+blueButton.addEventListener("click", blueMaker);
+function blueMaker() {
+  document.querySelector(".jumbotron").style.backgroundColor = "#588fbd";
+  document.querySelector(".buttons > a").style.backgroundColor = "#ffa500";
+  volBtn = document.querySelector(".buttons a:last-child");
+  volBtn.style.backgroundColor = "black";
+  volBtn.style.color = "white";
+}
+
+let orangeButton = document.getElementById("orangeBtn");
+orangeButton.addEventListener("click", orangeMaker);
+
+function orangeMaker() {
+  document.querySelector(".jumbotron").style.backgroundColor = "#f0ad4e";
+  document.querySelector(".buttons > a").style.backgroundColor = "#5751fd";
+  volBtn = document.querySelector(".buttons a:last-child");
+  volBtn.style.backgroundColor = "#31b0d5";
+  volBtn.style.color = "white";
+}
+
+let greenButton = document.getElementById("greenBtn");
+greenButton.addEventListener("click", greenMaker);
```

Comment:
Great work
There is some room for improvement, all items have same function style